css/styles.css

```diff
@@ -4,11 +4,15 @@
 body {
   margin: 0;
   font-family: Roboto, sans-serif;
-  font-size: 0.7vw;
+  font-size: 0.7rem;
```

Comment:
You'll see in a few places this change from `vw` units to `rem` for font sizes. Outside of a very few cases, I'd recommend not tying your font sizes directly to the window size (there are exceptions -- for example, I think keeping your `h1` size tied to the viewport width is a clever way to ensure that it is able to stay on one line at any screen size. However, even then, on small screens you get a really small title, and it might be better if it wraps anyway).